src/App.js

```diff
@@ -1,13 +1,73 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+import ToDoForm from './components/TodoComponents/TodoForm';
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor(){
+    super()
+    this.state = {
+      todo: [
+        {
+          task: 'Organize Garage',
+          id: 1,
+          completed: false
+        },
+        {
+          task: 'Bake Cookies',
+          id: 2,
+          completed: false
+        }
+      ],
+      newTask: '',
+    }
+  }
+
+  addItem = () =>{
+    const objItem = {
+      task: this.state.newTask,
+      id: Date.now(),
+      completed: false,
+    }
+    
+    this.setState({
+      todo: [...this.state.todo, objItem]
+    });
+    
+  }
+  handleInput = event => {
+   
+    this.setState({
+      newTask: event.target.value
+    })
+  }
+  removeTask = () =>{
+    this.setState({
+      todo: this.state.todo.filter((complete) =>{
+        return complete.completed === false
+      })
+    })
+  }
+  handleClick = (givenId) => {
+    this.setState({
+      todo: this.state.todo.map(toDoObj =>{
+        if(givenId === toDoObj.id){
+          toDoObj.completed = !toDoObj.completed
+        }
+        return toDoObj;
```

Comment:
remember our zoom about this: it may be a good idea to add a comment above each line, explaining what the code does